ansible_collections/arista/cvp/docs/release-notes/v3.x.md

```diff
@@ -16,10 +16,27 @@
 
 ## Release 3.11.0
 
+Full release note available on [github](https://github.com/aristanetworks/ansible-cvp/releases/tag/v3.11.0)
```

Comment:
We should also callout new ansible-core support.

Reply:
yes, already have that in the requirements section below